arbeitszeit_flask/filters.py

```diff
@@ -1,19 +1,21 @@
 import os
+from importlib import resources
 from typing import Callable, Dict
 
 from markupsafe import Markup
 
-ICON_PATH = "arbeitszeit_flask/templates/icons"
+ICON_PACKAGE = "arbeitszeit_flask.templates.icons"
 
 
-def file_reader(file_path: str) -> str:
-    with open(file_path, "r") as file:
-        return file.read()
+def icon_file_reader(file_name: str) -> str:
+    traversable = resources.files(ICON_PACKAGE).joinpath(file_name)
```

Comment:
Because the instance, returned  in `resources.files(ICON_PACKAGE).joinpath(file_name)`, is described as such in the docstring of .joinpath(...) and returns a type `Traversable`, so I thought, lets keep the naming generic (universal), hence the variable name `traversable`:

```
(method) def joinpath(*descendants: str) -> Traversable
Return Traversable resolved with any descendants applied.
```

Reply:
I agree, I will change the name.